Build/Build.cs

```diff
@@ -128,6 +131,7 @@ class Build : NukeBuild
                 )
                 .EnableNoLogo()
                 .EnableNoRestore()
+                .SetVersion(SemVer)
```

Comment:
Does this have any advantage when compiled with a pre-release version instead of 7.0.0 in this case?

I know that the actual nuget version have to be versioned correctly, but does this also apply to the .dll itself? Or doesn't this mater in any way?

Reply:
I'd like to keep the DLL versions in sync with the package versions, yes.

Reply:
So.. with v6 there are a lot of breaking changes as it seems.. but I figured out (with help from [here](https://github.com/GitTools/GitVersion/discussions/4181) what the problem was.